neuralmonkey/dataset/helpers.py

```diff
@@ -100,11 +100,23 @@ def _preprocessed_datasets(
         name = get_first_match(PREPROCESSED_SERIES, key)
         preprocessor = cast(DatasetPreprocess, series_config[key])
 
-        if isinstance(dataset, Dataset):
+        if isinstance(dataset, LazyDataset):
+            # Store the preprocessed series in the right place in lazy dataset:
+            # Note the `None` in the assignment. Since there is no single
+            # source series ID, we just don't use it.
+            if name in dataset.preprocess_series:
+                raise ValueError(
+                    "Series already exists: {}".format(name))
```

Comment:
Series (pl.) -> exist

Reply:
tady je to singulár

https://ell.stackexchange.com/questions/23268/is-series-plural-or-singular